shared/constants/wallets.tsx

```diff
@@ -293,6 +293,7 @@ export const currencyResultToCurrency = (w: RPCTypes.CurrencyLocal) =>
 
 const _defaultPaymentCommon = {
   amountDescription: '',
+  assetCode: '',
```

Comment:
Which asset is this referring to? Could this field have a more specific name

Reply:
The destination asset. I suppose `destinationAssetCode`, `destinationCode`, etc. could be more specific. I went with just `assetCode` because the payment object seems to consider all properties not prefixed with `source` to be referring to the destination – e.g., `issuerAccountID` and `issuerDescription` refers to the destination asset's issuer's account id and description respectively. The corresponding `source` properties are `sourceAccountID` and `sourceIssuer`.

Since payment can refer to both a direct payment or a path payment, we should probably decide on some sort of convention for names like these

Reply:
Agreed about making a convention for them. For now a comment in the avdl is fine.